PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -189,10 +189,9 @@ class HTTPClientTests: XCTestCase {
             response.error = error
             return response
         }
-        self.client.perform(request, authHeaders: [:]) { (status, data, responseError) in
-            if let responseNSError = responseError as NSError? {
+        self.client.perform(request, authHeaders: [:]) { (status, result) in
+            if let responseNSError = result.error as NSError? {
                 successFailed.value = (status.isServerError
```

Comment:
I know this variable isn't from this PR, but this name confuses me a lot - what does `successFailed` represent? 
Is it something like `operationFailed` or `requestFailed`?

Reply:
Changed! And I also changed the `expectToEventually` because it would wait _forever_ if the test fails instead of failing right away after this is evaluated.